a = 3
assert(a > 2)
print('断言成功，程序继续执行下去')


b = 5
assert(b == 4)
print('断言失败，程序报错')
